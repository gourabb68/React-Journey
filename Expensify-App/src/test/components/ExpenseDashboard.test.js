import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboradPage, {ExpenseDashboardd} from '../../components/ExpenseDashboardPage';


test('should render expense Dashboard',()=>{
    const wrapper=shallow(<ExpenseDashboradPage/>)
    expect(wrapper).toMatchSnapshot();
})
