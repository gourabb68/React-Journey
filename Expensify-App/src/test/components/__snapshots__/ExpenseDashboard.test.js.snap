// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense Dashboard 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
